test/NHSD.BuyingCatalogue.Documents.API.Tests/AzureBlobStorageTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using FluentAssertions;
+using Moq;
+using NHSD.BuyingCatalogue.Documents.API.Storage;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Documents.API.Tests
```

Comment:
I assume this is Unit Tests? Could we be more specific in the name of the project? e.g. NHSD.BuyingCatalogue.Documents.API.UnitTests